internal/conditions/cerbos_lib_benchmarks_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021-2022 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package conditions
```

Comment:
I made `program` private to avoid issues with reusing a program instance, but this meant that I had to move the benchmarks to a whitebox test - they deliberately reuse the program instance in order to only measure the expression evaluation time without the (small) overhead of constructing the program.